docs/content/concepts/middleware/_index.md

```diff
@@ -167,15 +167,21 @@ spec:
 In this configuration, the containers will only be unwound into separate Container nodes when the Pod event has `action: "update"`. Events with other actions (like "delete" or "create") will be processed normally but won't trigger the unwinding of container information.
 
 
-### Map
+### JQ
 
-The **map** middleware component can be used to remap an incoming insert/update/delete from a source to a different insert/update/delete for another element.
+The **jq** middleware component can be used to transform an incoming source change into a different shape or project it out into multiple changes. It is built on the popular [jq](https://jqlang.org/) tool.
 
-The configuration properties for the **map** component are as follows
+#### Links
+
+- [JQ Manual](https://jqlang.org/manual/)
+- [JQ playground](https://play.jqlang.org/)
+
+
+The configuration properties for the **jq** component are as follows
 
 | Property | Description |
 | - | - |
```

Comment:
I recall that there is a configuration called `halt_on_error` in the code. Should we add a brief sections that explains how to override this config?